### PR TITLE
Changed the labels to be less confusing (CU-2mdcnzt)

### PR DESCRIPTION
**Pull request by PeterScholtens**

### CODE REVIEW

**Comment by trisapeace** [comment]
Task linked: [CU-2mdcnzt Change labels on the PA](https://app.clickup.com/t/2mdcnzt)

**Comment by PeterScholtens** [comment]
I did not run any test cases as i was just changing text.

**Comment by PeterScholtens** [comment]
Deployed it to dev and then checked the Twilio Number Purchasing page and the labels had been changed to what I wanted them to be. It worked like it should. Test passed

File: CHANGELOG.md [hunk 1]
```diff
@@ -14,6 +14,7 @@ the code was deployed.
 ### Changed
 - Sensor Provisioning Guide step 6 to include maching door sensor address. 
 - Typo on the Particle CLI command and its corresponding image. 
+- Changed labels on the Twilio Number Purchasing page for more clarification.
```
Comment (trisapeace) [inline CHANGELOG.md]: Please include the ClickUp `CU-code` in brackets before the period like in the other repositories.

Comment (PeterScholtens) [reply CHANGELOG.md]: For sure

**Review by trisapeace — APPROVED** [review]
Looks great!

Can you please squash your three commits together and then use the Rebase & Merge button to merge your branch into `main`.

Then yes, could you please go through the deployment process in the `README`. That would be great. Thanks!

**Pull request merged**